### PR TITLE
Covid-19 Statistics web app(Vaccination stat only)

### PR DESCRIPTION
**Pull request by tylher**
- Set up tools for the project
- install all dependencies for the project
- Write reducer functions and actions to dispatch in component
- Fetch data from an API and manipulate it according to the data needed for the interface
- Write tests for components and pure functions
- set up linters 
- Follow design specifications to style pages
- Document project in a README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,117 @@
...
+- Technologies used:
+  > React,
+  > Jest,
+  > Redux,
+  > JSON,
+  > Gitflow
+  > rest API
+
+## live
+[View live demo here]()
```
Comment (SajjadAhmad14) [inline README.md:19]: Please deploy the project and add a live link in README.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useDispatch } from 'react-redux';
+import { Route, Routes } from 'react-router';
+import './App.css';
+import Continents from './components/continents/Continents';
+import Countries from './components/countries/Countries';
```
Comment (SajjadAhmad14) [inline src/App.js]: There is an error while compiling the app. Kindly fix the error
app-error
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Damola, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 2 above)
Comment (tylher) [reply src/App.js]: Thank you @SajjadAhmad14 , I have resolved this

Comment (mateo951) [reply src/App.js]: Addressed ✔

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @tylher , 

Well done resolving all the requested changes 💪😎
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**